src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,7 +1,15 @@
 package mate.academy
 
+import kotlin.system.exitProcess
+
 fun greetUsers() {
-    // implement function
+    println("Please enter a name (or press Enter to quit): ")
+    val input = readLine()
+    if (input == " " || input == "") {
```

Comment:
Please check these methods for code optimization
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/is-null-or-empty.html
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/if-empty.html